test/e2e/README.md

```diff
@@ -25,9 +59,9 @@ properly picked up. Additionally, a module called `sinspqa` lives in
 to house any functions/classes that might be useful across multiple tests. The
 dockerfile for this image can be found under `containers/tests.Dockerfile`.
 
-## Running the tests
-An `e2e-tests` target has been added. It requires the `BUILD_BPF` option to be
-set.
+### Running the tests
```

Comment:
Similar to the other comment, all commands in one concise blob may be welcomed in case you are in a rush :upside_down_face: and can't read all details.

```sh
cd /tmp;
git clone https://github.com/falcosecurity/libs.git;
cd libs;
mkdir -p build;
cd build;
cmake -DCREATE_TEST_TARGETS=ON -DUSE_BUNDLED_DEPS=ON -DBUILD_BPF=ON -DBUILD_DRIVER=ON ..;
# Ensure you did run `pip3 install test/e2e/tests/commons` step from the libs root directory
make e2e-tests # run as non-root user
```